### ISSUE TITLE
BibEdit: Improve suggestion output

### ISSUE THREAD

**Issue by audub** [tind/invenio-fork#18]
From Alex via support

Can we increase the design of the actual suggestion box? I think we can make it a lot prettier. This is not a "need to have" from my side, but very "nice to have". I guess it would depend on how much javascript trouble this would cause us. Some suggestions:

- Make it a pop-up box instead of within the same field (currently looks very cramped/crowded). This would allow for more "space".
- differentiate between Authors and Authorities (I think people would always prefer an authority if there is one available)
- highlight author names in bold, to distinguish between important and less important information. Makes it quicker to skim through the list and look for the right author.

![screen shot 2015-02-18 at 10 23 22](https://cloud.githubusercontent.com/assets/2854307/6306396/902d00d8-b933-11e4-9044-89bb7508bc6e.png)

Note from Audun: 
We will discuss and agree on a solution before we start working on this.

**Comment by PXke** [comment]
Something implemented,  thanks to check it, should be online at 00h00 the 24 of march on hr-mig2

**Comment by nietzold** [comment]
Cool. Some comments:1. Can you make the tables width fixed? I.e. same width
as all other tables.

[image: Inline image 1]

2. Knowledge base = authors in the database that match the entry? or? does
this have something to do with BibKnowledge?

3. Change the background color of selected elements to RGBA(41, 128, 185,
0.2)

[image: Inline image 2]

4. When no entries are available, there is a little blue line. Can this be
removed? Or, perhaps the table can show anyway, just displaying "no entries
found"?

[image: Inline image 5]

5. Change design of "close" button to a more similar style as other Invenio
buttons, such as this one (doesn't to as big as this one,  but at least the
same colors):

[image: Inline image 6]



best regards
Alexander Nietzold
tlf: +47 926 42 726

**Comment by audub** [comment]
@nietzold the images does not seem to work on github

**Comment by PXke** [comment]
Indeed I see no picture, I will do the changes :)

**Comment by nietzold** [comment]
- [ ] Can you make the tables width fixed? I.e. same width
as all other tables.

![screen shot 2015-03-24 at 19 45 32](https://cloud.githubusercontent.com/assets/4989971/6824296/1302c27a-d2ae-11e4-9785-da7651f88ef4.png)


- [ ] Knowledge base = authors in the database that match the entry? or? does
this have something to do with BibKnowledge?

- [ ] Change the background color of selected elements to RGBA(41, 128, 185,
0.2)

![screen shot 2015-03-24 at 20 12 08](https://cloud.githubusercontent.com/assets/4989971/6824299/1a52c962-d2ae-11e4-9f47-00f20419ea5a.png)


- [ ] When no entries are available, there is a little blue line. Can this be
removed? Or, perhaps the table can show anyway, just displaying "no entries
found"?


![screen shot 2015-03-24 at 20 15 38](https://cloud.githubusercontent.com/assets/4989971/6824311/2e276402-d2ae-11e4-9499-1d42dcf9452a.png)


- [ ] Change design of "close" button to a more similar style as other Invenio
buttons, such as this one (doesn't to as big as this one,  but at least the
same colors):

![screen shot 2015-03-24 at 20 17 56](https://cloud.githubusercontent.com/assets/4989971/6824302/1fc8c036-d2ae-11e4-8b7f-7bb213cdeb95.png)

**Comment by PXke** [comment]
Much better :) Thank you

**Comment by audub** [comment]
Regarding your question about bib knowledge:

Guillaume and I have discussed it, and we are not 100% sure about the behavior, but you can create tags in bib knowledge which is displayed here. If you dont, invenio will generate a tag based on what you entered. But if all of this is bib knowledge, or if it is based on something else is still not 100 % clear.

**Comment by nietzold** [comment]
Why is bibknowledge/authors not showing on caltech? I only get authorities, no authors. Don't know if that explains anything?

**Comment by audub** [comment]
Not sure, @PXke ?

**Comment by PXke** [comment]
Not sure. I will need an exploration in the guts of invenio.

**Comment by blixhavn** [comment]
This issue was moved to tind/invenio#2
